DQM/SiStripMonitorDigi/interface/SiStripMonitorDigi.h

```diff
@@ -196,7 +199,7 @@ class SiStripMonitorDigi : public DQMOneEDAnalyzer<edm::one::WatchLuminosityBloc
   bool globalswitchNDigisFEDID;
 
   //  int xLumiProf;
-  bool isStableBeams;
+  mutable bool isStableBeams;
```

Comment:
I'd think this should be moved to the lumi cache instead. Otherwise it could be that one EDM stream starts processing the next lumi (setting `isStableBeams = false`) before some other stream has called `analyze()` for this module.

Reply:
Or maybe a bigger problem is that the `globalBeginLuminosityBlock()` can be called for lumi N+1 before the `globalEndLuminosityBlock()` for lumi N, which would re-set `isStableBeams=false` also for lumi N regardless of it being set to `true` in `analyze()`

Reply:
@makortel that's a good point. I am updating the PR. Thanks.